test/api/v01/request_helper.rb

```diff
@@ -26,6 +26,7 @@ def wait_status(job_id, status, options)
       get "0.1/vrp/jobs/#{job_id}.json", options
 
       assert_equal 200, last_response.status, last_response.body
+      assert_includes ['queued', 'working', 'completed'], JSON.parse(last_response.body)['job']['status']
```

Comment:
Can we use a similar check in `wait_status_csv` or is there no risk there?

Reply:
You are right, we can add it in `wait_status_csv` too.